### ISSUE TITLE
Add remark-unwrap-images to @gridsome/transformer-remark default plugins

### ISSUE THREAD

**Issue by missmatsuko** [gridsome/gridsome#700]
## Summary
`remark-unwrap-images` remark plugin removes paragraph tags that wrap images.

### Basic example
[GitHub for remark-unwrap-image](https://github.com/remarkjs/remark-unwrap-images)

### Motivation
It seems like a good default plugin to add since paragraphs shouldn't be wrapping images.

**Comment by midlantica** [comment]
Is there no way to break this curse? :P

**Comment by missmatsuko** [comment]
Right? I tried adding it myself but couldn't get it working. I think there's some conflict between when images get unwrapped and when they get replaced with g-image.

**Comment by kevinzweerink** [comment]
Also interested in support for this—didn't work for me either when I tried to install it myself. Currently using Vue lifecycle hooks to add classes to images within paragraphs, but it doesn't feel like a good solution.

**Comment by jamesjlyons** [comment]
I'm also trying to get this to work. I thought this might have to do with #707 and the new vue-remark, but it seems like a separate issue for me since clearing my cache and trying multiple configurations in my gridsome.js hasn't helped.

**Comment by AchrafTani** [comment]
@missmatsuko Isn't there anything new about this? I'm also having the same problem ? 
Thanks.

**Comment by sebtiz13** [comment]
hello remark-unwrap-image can't unwrap the image transformed by gridsome because it's convert in  `html` block.

https://github.com/gridsome/gridsome/blob/master/packages/transformer-remark/lib/plugins/image.js#L42-L45

But this plugin search block of type `image` or `imageReference`.
I think it's convert in `html` block because have "prerendered" img and noscript element.

A good solution can be add an option like "unwrapImage".
I have try this solution with simple implementation of plugin.
https://github.com/sebtiz13/gridsome/commit/f6f25e9fd2cd6d0a50dad0cd0439b6d41727c318

In waiting of real solution, i have create an hacky plugin to unwrap the outputted html image without modify package.

https://gist.github.com/sebtiz13/22c2765699773fcc45e017d521704c71

**Comment by alexwidua** [comment]
In case someone runs into the same issue, I did a quick Gridsome implementation of [gatsby-remark-unwrap-images](https://github.com/cedricdelpoux/gatsby-remark-unwrap-images), you can find the repo here: [gridsome-remark-unwrap-images](https://github.com/alexwidua/gridsome-remark-unwrap-images).

It basically the same approach @sebtiz13 did. There is one caveat that images have to be placed in a seperate line with one line break to the next paragraph, otherwise the image still gets wrapped in a `<p>` tag (see [example](https://github.com/alexwidua/gridsome-remark-unwrap-images#warning-caveat)).

**Comment by sebtiz13** [comment]
@alexbrown for your solution more reliable :pray: 

But personally I prefer if it's can be an options of [@gridsome/transformer-remark](https://gridsome.org/plugins/@gridsome/transformer-remark) so I make PR #1290 for that with more possibilities to cover #1195

**Comment by alexwidua** [comment]
Yeah the plugin is more like a band-aid than a real solution for the problem. Thanks for the PR, this would be a neat feature (especially wrapping images in a different tag)
